libraries/compute-utils/function-views/src/rich-function-view.ts

```diff
@@ -879,6 +1055,8 @@ export class RichFunctionView extends FunctionView {
   }
 
   private hideOutdatedOutput() {
+    if (this.keepInputs())
```

Comment:
Hiding output tabs creates problems somewhere?
If you want to prevent UI "blinking", I suggest:

1. avoid adding one more tag
2. add ui.setUpdateIndicator for output tabs while output is recalculated

Reply:
Don't get it.

If the output result is used to tweak inputs - why couldn't we wait for the function run to end? Then see outputs as usual, analyze them and tweak inputs

Btw, it should be called `keepOutputs`, I think

Reply:
Aaah, got it. 

Well, this case should be resolved in different way: we should mark the output as outdated (it is very important for consistency and not-confusing users), but should not hide it. Let's keep your solution for now - later I will introduce better approach and will remove the tag. 

Is it OK?